app/controllers/reviews_controller.rb

```diff
@@ -161,6 +161,11 @@ def vote_down
       current_user.vote_against(@review)
     end
 
+    if ((@review.votes_against > -9) && @review.comment != "")
+      @review.deleted = true
+      @review.save
```

Comment:
`.update_attribute` doesn't trigger validations, just use `.update(:deleted => false)`.

however, in practice if `@review.save` fails due to validations, you'll want access to `@review.errors` which is nice